extensions/schema-compare/src/test/schemaCompare.test.ts

```diff
@@ -160,7 +160,7 @@ describe('SchemaCompareMainWindow.execute', function (): void {
 		await result.execute();
 
 		//Generate script button and apply button should be disabled for dacpac comparison
-		should(result.verifyButtonsState(false, false)).equal(true);
+		should(result.verifyButtonsState(true, true, true, true, true, false, true, true, false, false)).equal(true);
```

Comment:
Is there a way to provide names to these magic constants in TyprScript?

Reply:
sadly no, but you can do it indirectly by defining an interface: https://stackoverflow.com/a/30734743

Reply:
Hmm ok. I will keep it as is for now then.

Reply:
I actually think the interface is a great idea to make the test code more readable, especially if you pull the `verifyButtonsState` class out into a test-only subclass like the other comments recommend.

Reply:
Done.